target/linux/ramips/mt7621/base-files/lib/upgrade/platform.sh

```diff
@@ -150,6 +150,12 @@ platform_do_upgrade() {
 	ubnt,edgerouter-x-sfp)
 		platform_upgrade_ubnt_erx "$1"
 		;;
+	zte,mf286c3)
+		echo "Upgrading zte,mf286c3"
+		CI_UBIPART="ubi"
+		CI_KERNPART="firmware"
```

Comment:
Shouldn't the "firmware" partition be renamed as "kernel" then, and the upgrade process handled by generic "nand_do_upgrade"?

Reply:
No, this should be removed altogether and device entry moved to section with generic "nand_do_upgrade".